app/helpers/caption_helper.rb

```diff
@@ -0,0 +1,53 @@
+module CaptionHelper
+
+  require 'webvtt'
+
+  def caption_contents caption_url
+    caption_fullurl = File.join(root_url,caption_url)
+    caption_file = open(caption_fullurl,"rb")
+    
+    if caption_file.is_a?(StringIO)
+      # This happens when file is <10KB: it opens as a string. So write it back to a tmp file.
+      tmp_vtt_file = Tempfile.new('ddr-vtt-#{Time.now.utc}')
```

Comment:
Double quotes needed for string interpolation to work: `"ddr-vtt-#{Time.now.utc}"`

Reply:
@corylown Excellent catch--thanks!